### ISSUE TITLE
finish QC for CanFam

### ISSUE THREAD

**Issue by petrelharp** [popsim-consortium/stdpopsim#1242]
**PR for new species:** #375 - this was already QC'ed in #652 but before we had checks in there for recombination and mutation rates (see other species) - so, most of these things below are probably done already!

If you volunteer to QC this species, please use the checklist below.
While this list is intended to be comprehensive, it may not be exhaustive.
Where relevant, the QC reviewer should identify that parameter values match
those given in the linked citation(s).

The QC reviewer should start a pull request that fills out the test stubs
with independently obtained values. The reviewer may look at the python code
for rationale provided in comments, but should ignore the actual code
as much as possible - comments in the code should give enough information
that it's obvious how to get the correct value from the provided references.
(In particular, we shouldn't copy-paste the value from the code into the test!)

- [x] Recombination rate.
  - This might be genome-wide, or per-chromosome. Both are fine.
  - Check there's a comment describing where it came from, and/or how calculated.
  - From a publication? Check the value(s) match the publication.
  - Calculated somehow? Average over a recombination map? Redo the calculation.
- [x] Mutation rate.
  - This might be genome-wide, or per-chromosome. Both are fine.
  - Check there's a comment describing where it came from, and/or how calculated.
  - From a publication? Check the value(s) match the publication.
  - Calculated somehow? Redo the calculation.
- [x] Recombination map (if present).
  - Does it match the assembly? Liftover is fine, if clearly stated.
  - Is the description/long_description a good summary of how the map was created?
- [x] Population size.
- [x] Generation time.

For each citation, check:
- Doi link.
- Is publication a preprint? Is there a peer-reviewed publication instead?
- Is the year correct.
- Is the author correct (spelling, accents/ligatures/etc.)

Citations are required for:
- [x] Genome reference assembly.
- [x] Mutation rate.
- [x] Recombination rate.
- [x] Recombination map(s) (if relevant).
- [x] Population size.
- [ ] Generation time.

The final PR should:
- [x] fill out the test stubs
- [x] delete the `pytest.mark.skip` lines that make the tests not run
- [x] make sure they pass, talking to the original author to figure out discrepancies

**Comment by grahamgower** [comment]
See also #540. Is it mainly the tests for recombination/mutation rates that need doing @petrelharp?

**Comment by petrelharp** [comment]
Yeah, I think that's it.

**Comment by petrelharp** [comment]
I'll start doing this one.

**Comment by petrelharp** [comment]
The mutation rate citations are:
```

_SkoglundEtAl = stdpopsim.Citation(
        # Ancient wolf genome reveals an early divergence of domestic dog
        # ancestors and admixture into high-latitude breeds.
        author="Skoglund et al.",
        year=2015,
        doi="https://doi.org/10.1016/j.cub.2015.04.019")
_FranzEtAl = stdpopsim.Citation(
        # Genomic and archaeological evidence suggest a dual origin of
        # domestic dogs.
        author="Franz et al.",
        year=2016,
        doi="https://doi.org/10.1126/science.aaf3161")
```

**Comment by petrelharp** [comment]
It all looks good, except there's no citation for generation time, only the comment:
```
        # Everyone uses 3 years for generation time because everyone else uses it.
        # It's likely higher, at least in wolves:
        # https://academic.oup.com/mbe/article/35/6/1366/4990884
        # Reasoning behind a generation time of 3 years:
        # Consider two use cases for CanFam simulations:
        # (1) for domestic dog simulations, and (2) for wolf+dog simulations
        # (or ancestral dogs).
        # In case (1), maybe 3 year generations are more appropriate because of human
        # intervention in breeding. In case (2), you might want to match what other
        # studies have done (thus using 3 year generations), or you might want to
        # consider what is known about modern wolves.
```
I'm going to let this slide.

**Comment by petrelharp** [comment]
Closed in #1258
